src/plugins/vis_augmenter/public/vega/helpers.ts

```diff
@@ -322,3 +327,24 @@ export const addPointInTimeEventsLayersToSpec = (
 
   return newSpec;
 };
+
+export const interactionHandlers = {
+  ['visAugmenter.handlePointInTimeClick']: (event: any, item: any) => {
+    if (isPointInTimeView(item)) {
+      alert('clicked annotation');
+    }
+  },
+};
+
+export const addPointInTimeInteractionHandlers = (spec: any) => {
+  spec.interactionHandlers = [
+    {
+      event: 'click',
+      expressionFunction: 'visAugmenter.handlePointInTimeClick',
+    },
```

Comment:
Recommend making the interaction handlers typesafe. Also, `expressionFunction` field could be renamed to just `function` or something else, since it is not an expression function.